.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -1,10 +1,23 @@
 # Description
 
-Please include a summary of the change and which issue is fixed. 
+[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)
+
+## Types of changes
+
+[//]: # (What types of changes does your code introduce to Lingui?)
+[//]: # (_Put an `x` in the boxes that apply_)
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update
```

Comment:
Maybe it would be good to add "Chore" as a type of change 🙂

Reply:
Good point 🙂

Reply:
Or maybe we can leave here only these essential types. Detailed type anyway will be specified in the PR title, it's not required to select these checkboxes in this case (chore, ci, refactoring, etc)

Reply:
I think we can leave only the essential types here for now